### PR TITLE
Make error messages more helpful

### PR DESCRIPTION
**Pull request by erino**
Show who the publisher should contact if they can't access a publishing application.

https://trello.com/c/bOxIvf1L

### CODE REVIEW

File: app/models/organisation.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 class Organisation < ActiveRecord::Base
   include ActiveModel::ForbiddenAttributesProtection
 
+  GDS_CONTENT_ID = "af07d5a5-df63-4ddc-9383-6a666845ebe9".freeze
```
Comment (cbaines) [inline app/models/organisation.rb]: Maybe `GDS_ORG_CONTENT_ID`?

Comment (cbaines) [reply app/models/organisation.rb]: Just because I got confused for a second.

File: app/views/root/signin_required.html.erb [hunk 2]
```diff
@@ -1,16 +1,24 @@
-<% content_for :title, "Sorry, you don't have permission to access " + application_name %>
+<% content_for :title, title %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% if gds_only_application_and_non_gds_user? %>
+    <% else %>
```
Comment (cbaines) [inline app/views/root/signin_required.html.erb]: Is the above part of the `if` meant to be empty?

File: app/helpers/root_helper.rb [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+  def gds_only_application_and_non_gds_user?
+    @application.gds_only? && !current_user.belongs_to_gds?
+  end
+
+  def title
+    if @application.blank? || gds_only_application_and_non_gds_user?
+      "You don’t have permission to use this app."
     else
-      "this application"
+      "You don’t have permission to sign in to " + @application.name + "."
```
Comment (cbaines) [inline app/helpers/root_helper.rb]: `in to #{@application.name}."` might be neater and more consistent .

Comment (erino) [reply app/helpers/root_helper.rb]: 👍

File: app/views/root/signin_required.html.erb (hunk 2 above)
Comment (erino) [reply app/views/root/signin_required.html.erb]: Yes, fixed in 9cae5e4b1cbe9786deb7b44c7a594904900ef85a

File: app/models/organisation.rb (hunk 1 above)
Comment (erino) [reply app/models/organisation.rb]: 👍

**Pull request merged**